ui/src/core_components/other/CoreReusable.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+	<div class="CoreReuse">
+		<component :is="vnode" />
```

Comment:
🚨🚨🚨 I think the whole thing will blow up when using _Sidebar_, which is one of the main use cases.

Reply:
Yes, it exploded for sidebar.